examples/typescript/src/App.tsx

```diff
@@ -1,14 +1,14 @@
-import React, { useState } from 'react'
 import {
-  GraphQLClient,
+  APIError,
   ClientContext,
-  useQuery,
-  useMutation,
+  GraphQLClient,
   useManualQuery,
-  APIError
+  useMutation,
+  useQuery
 } from 'graphql-hooks'
+import React, { useState } from 'react'
 
-interface Post {
+interface PostData {
```

Comment:
This renaming was needed to avoid an eslint warning (but maybe error on CI).
All the rest is formatting.